common/djangoapps/third_party_auth/provider.py

```diff
@@ -39,7 +39,7 @@ def enabled(cls):
     @classmethod
     def accepting_logins(cls):
         """Returns list of providers that can be used to initiate logins currently"""
-        return [provider for provider in cls.enabled() if provider.accepts_logins]
+        return [provider for provider in cls.enabled() if provider.display_for_login]
```

Comment:
It's certainly doable, but we'd likely be changing most, if not all, instances of calls to `accepting_logins` to calls to `displayed_for_logins`.